application-task-default/src/main/java/com/xwiki/task/internal/TaskMacroUpdateEventListener.java

```diff
@@ -132,7 +132,7 @@ private boolean maybeHandleFoldEvent(XWikiDocument document, XWikiContext contex
                     updateTaskPages(context.getWiki().getDocument(lastFoldDocumentReference, context), context);
                 } catch (XWikiException e) {
                     logger.warn(EXCEPTION_DOCUMENT_RETRIEVAL, lastFoldDocumentReference,
-                        ExceptionUtils.getRootCauseMessage(e));
+                        ExceptionUtils.getStackTrace(e));
```

Comment:
Even if you format the error message with a bunch of parameters, you can pass the Throwable to the logger.error method as the final parameter and the stack trace will be printed.

See https://github.com/xwiki/xwiki-commons/blob/master/xwiki-commons-core/xwiki-commons-logging/xwiki-commons-logging-common/src/main/java/org/xwiki/logging/internal/AbstractLogger.java#L61-L62